MissedRevenueHeatMapGenerator.py

```diff
@@ -29,7 +29,7 @@ def calculate_run_rate(self, sku, branch, journal, period):
         #return run rate for a period
         pass
     
-    def calculate_missed_revenue(self, sku, branch, retail_price, qty_of_lost_sales):
+    def calculate_missed_revenue(self, sku, branch, retail_price, journal):
```

Comment:
Go draw a box on paper. Draw the inputs as arrows and the outputs as arrows. Now describe the shape of each and you will see your mistake.